docs/guides-publish.md

```diff
@@ -35,104 +37,98 @@ This app has 3 environments defined:
-
-<span>![*](/docs/assets/check.svg) Generate application artifact.</span>
-
-<span>![*](/docs/assets/check.svg) Publish the app to the **rinkeby** environment.</span>
+- Apply version bump (major).
+- _Compile_ and _deploy_ the app's contract (by default the output lives in `build`)
+- _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)
+- Generate application artifact.
+- Publish the app to the **rinkeby** environment.
+- Upload the app's frontend to the IPFS server.
 
 Sample output:
 
 ```sh
- > npx aragon apm publish major --environment rinkeby
-
- ✔ Successfully published app.open.aragonpm.eth v1.0.0:
- ℹ Contract address: 0xE636bcA5B95e94F749F63E322a04DB59362299F1
- ℹ Content (ipfs): QmR695Wu5KrHNec7pRP3kPvwYihABDAyVYdX5D5vwLgxCn
- ℹ Transaction hash: 0x3d752db29cc106e9ff98b260a90615921eb32471425a29ead8cbb830fb224d8
+main     |   App name:          app.aragonpm.eth
+main     |   Initial version:   1.0.0
+main     |   Manager address:   0x5Ddb5ec4fF143fDaBCCD0a47F30FF2ce319C2a01
+main     |   Contract address:  0x2e25c8F88c5cCcbC9400e5bc86cF9C58C7604327
+main     |   ContentURI:        QmTBapuxxzFHzxdbxxZUmdEmyRxQohxoK7qiXiJ6id36tu
+main     | 
+main     |   http://localhost:8080/ipfs/QmTBapuxxzFHzxdbxxZUmdEmyRxQohxoK7qiXiJ6id36tu
+main     | 
+main     | 
+main     |   Tx sent
+main     | 
+main     |   Tx hash:  0x954fa737215152d2c9be5893bcda91cf889b30dd5e26bada10b657e87b89692f
+main     |   
+main     | 
+main     |   Tx mined
+main     |   
+main     |   Status:        Success
+main     |   Block number:  78
+main     |   Gas used:      960591
 ```
 
 > **Note**<br>
 > You can also deploy a major version with only frontend changes by passing `--only-content`.
 >
-> You can also just generate artifacts file without publishing by passing `--only-artifacts`.
->
 > The contract location is defined in `arapp.json` under `path`.
 
-## Publish a minor or patch version: content only
 
-Command:
+## Task syntax and options
+
+The publish task has the following syntax:
 
 ```sh
-npx aragon apm publish patch --environment rinkeby
+npx buidler publish [global options] <bump> [task options]
 ```
 
-This will:
-
-<span>![*](/docs/assets/check.svg) Apply version bump (patch).</span>
+Where `global options` are Buidler's [global options](https://buidler.dev/getting-started/#quick-start) and `bump` is the version bump (either `major`, `minor` or `patch`) or the semantic version. E.g. `minor` would increase version `1.2.0` to `1.3.0`.
 
-<span>![*](/docs/assets/check.svg) _Build_ the app's [frontend (by default the output lives in `dist`)](#building-frontends)</span>
+The following task options are available:
 
-<span>![*](/docs/assets/check.svg) Generate application artifact.</span>
-
-<span>![*](/docs/assets/check.svg) Publish the app to the **rinkeby** environment.</span>
-
-Sample output:
+- `contract`: Contract address previously deployed.
```

Comment:
```suggestion
- `contract`: Address of the app's deployed smart contract
```

Note that you technically do not need to previously deploy this—you can counterfactually assign these addresses.